src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form_big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label_big">
+        <input
+          data-qa="keypress"
+          type="text"
+          placeholder="Try “Los Angeles”"
+          class="search-form__input search-form__input_big"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form_small"
+      data-qa="small"
+    >
+      <label class="search-form__label search-form__label_small">
+        <input
+          data-qa="hover"
+          type="text"
+          placeholder="Try “Los Angeles”"
+          class="search-form__input search-form__input_small"
+        />
+      </label>
+    </form>
   </body>
```

Comment:
The semantic structure of the HTML document could be improved. Consider using semantic HTML5 elements like `<header>`, `<main>`, `<section>`, `<article>`, `<aside>`, `<footer>`, etc. These elements provide additional information about your document to both the browser and the developer.